contrib/buildahimage/upstream/Containerfile

```diff
@@ -43,7 +43,8 @@ RUN sed -e 's|^#mount_program|mount_program|g' \
         -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' \
         /usr/share/containers/storage.conf \
         > /etc/containers/storage.conf && \
-    chmod 644 /etc/containers/storage.conf
+    chmod 644 /etc/containers/storage.conf; \
```

Comment:
Please use `&&` instead of `;`, so that an error from `sed` won't just be ignored (the difference between `bash -c "false && echo ok"` and `bash -c "false; echo ok"`).  It's weird that we haven't been consistent about this between the various files.

Reply:
I saw that. I had mimicked the existing use from one of the three files. I thought it was suspicious, but assumed there may be a reason for it.

I've pushed a second commit that changes all the inconsistent `;` usage to be `&&` in all three of the containerfiles.

Reply:
Thanks!